db/migrate/20160513160206_initial_migration.rb

```diff
@@ -0,0 +1,30 @@
+class InitialMigration < ActiveRecord::Migration
+  def change
+    create_table :links do |t|
+      t.string :uri
+      t.integer :upvote_count
+      t.integer :downvote_count
+
+      t.timestamps null: false
+    end
+    
+    create_table :tags do |t|
+      t.string :name
+      t.integer :sequence
```

Comment:
To summarize: for possibly prioritizing links based on the priority of their tags (as I understood Dave's suggestion). Perhaps this shouldn't be included until we have a better idea of how exactly we'd implement it, though?